client/core/bookie.go

```diff
@@ -883,32 +883,6 @@ func (dc *dexConnection) refreshServerConfig() (*msgjson.ConfigResult, error) {
 		}
 	}
 
-	// Patch ConfigResponse.RegFees if no entry for DCR is there, meaning it is
-	// likely an older server using cfg.Fee and cfg.RegFeeConfirms.
-	if dcrAsset := cfg.RegFees["dcr"]; dcrAsset == nil {
-		dc.log.Warnf("Legacy server %v does not provide a regFees map.", dc.acct.host)
-		if cfg.RegFees == nil {
-			cfg.RegFees = make(map[string]*msgjson.FeeAsset)
-		}
-		if cfg.Fee > 0 {
-			cfg.RegFees["dcr"] = &msgjson.FeeAsset{ // v0 is only DCR
-				ID:    42,
-				Confs: uint32(cfg.RegFeeConfirms),
-				Amt:   cfg.Fee,
-			}
-		} else {
-			dc.log.Warnf("Server %v does not support DCR for registration", dc.acct.host)
-		}
-	} else {
-		if cfg.Fee > 0 && dcrAsset.Amt != cfg.Fee {
-			dc.log.Warnf("Inconsistent DCR fee amount: %d != %d", dcrAsset.Amt, cfg.Fee)
-		}
-		if dcrAsset.Confs != uint32(cfg.RegFeeConfirms) {
-			dc.log.Warnf("Inconsistent DCR fee confirmation requirement: %d != %d",
-				dcrAsset.Confs, cfg.RegFeeConfirms)
-		}
```

Comment:
Released clients will also warn about this because these dcr-specific fields will be zero with a server running the code changes in this PR, but the clients will still use any set DCR values in the `RegFees` map.